index.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link rel="stylesheet" href="styles.css">
+        <title>Practice Website</title>
+  </head>
+
+  <body>
+    <div class="top">
+      <div class="header">
+        <div class="header-logo">Header Logo</div>
+  
+        <div class="links">
+          <ul>
+            <li><a href="google.com">header link one</a></li>
+            <li><a href="google.com">header link two</a></li>
+            <li><a href="google.com">header link three</a></li>
+          </ul>
+        </div>
+      </div>
+  
+      <div class="spotlight">
+        <div class="left">
+          <div class="title">This website is awesome</div>
+          <p class="title-subtext">
+            This website has some subtext that goes
+            under the main title. It's a smaller font
+            and the colour is lower contrast.
+          </p>
+          <button class="sign-up">Sign up</button>
+        </div>
+  
+        <img class="right" src="leaves.jpg" alt="leaves">
```

Comment:
Maybe an [aside] semantic element would have been better here and it would remove the need for specific "left" and "right" classes